File: Examples/VisualBasic/Worksheets/Display/HideTabs.vb
Imports System.IO

Imports Aspose.Cells

Namespace Worksheets.Display
    Public Class HideTabs
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            ' Opening the Excel file
            Dim workbook As New Workbook(dataDir & "book1.xls")

            ' Hiding the tabs of the Excel file
            workbook.Settings.ShowTabs = False

            ' Shows the tabs of the Excel file
            ' Workbook.Settings.ShowTabs = True

            ' Saving the modified Excel file
            workbook.Save(dataDir & "output.xls")
            ' ExEnd:1
        End Sub
    End Class
End Namespace

File: Examples/CSharp/Worksheets/Display/HideTabs.cs
using System.IO;

using Aspose.Cells;

namespace Aspose.Cells.Examples.CSharp.Worksheets.Display
{
    public class HideTabs
    {
        public static void Run()
        {
            // ExStart:1
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);

            // Instantiating a Workbook object
            // Opening the Excel file
            Workbook workbook = new Workbook(dataDir + "book1.xls");

            // Hiding the tabs of the Excel file
            workbook.Settings.ShowTabs = false;


            // Shows the tabs of the Excel file
           // workbook.Settings.ShowTabs = true;

            // Saving the modified Excel file
            workbook.Save(dataDir + "output.xls");
            // ExEnd:1
        }
    }
}
